apps/src/lib/kits/maker/ui/DiscountCodeInstructions.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import View_Cart from "../../../../../../dashboard/app/assets/images/maker/View_Cart.png";
+import Add_to_cart from "../../../../../../dashboard/app/assets/images/maker/Add_to_cart.png";
+import Enter_Discount_Code from "../../../../../../dashboard/app/assets/images/maker/Enter_Discount_Code.png";
+
+const styles = {
+  title: {
+    fontSize: 32,
+  },
+  image: {
+    width: 400,
+  }
+};
+
+export default class DiscountCodeInstructions extends Component {
+  static propTypes = {
+    discountCode: PropTypes.string.isRequired,
+    discountComplete: PropTypes.bool.isRequired,
+  };
+
+  render() {
+    return (
+      <div>
+        <h1 style={styles.title}>{i18n.subsidizedPlaygroundKit()}</h1>
+        <h2>{i18n.discountCode({discountCode: this.props.discountCode})}</h2>
+        <div>
+          {i18n.discountCodeAnnouncement({discountAmount: this.props.discountComplete ? "$0" : "only $97.50 (including standard ground shipping)"})}
```

Comment:
We might actually be okay with this all being English only, but if we're going to do i18n we probably want "$0" and "only $97.50..." to be localizable.

Reply:
Confirmed with Poorva that we don't actually care about i18n for this, since it's for US teachers only. This means we can just embed all of our strings, as English, in the component itself.